readme.md

```diff
@@ -228,6 +228,11 @@ const queryString = require('query-string');
 
 queryString.stringify({foo: [1, 2, 3]}, {arrayFormat: 'comma'});
 //=> 'foo=1,2,3'
+
+queryString.stringify({foo: [1, null, '']}, {arrayFormat: 'comma'});
+//=> 'foo=1,,'
+// Note that typing information for null values are lost
+// and `.parse('foo=1,,')` would return `{foo: [1, '', '']}`.
```

Comment:
Readme and index.d.ts should be in sync.

Reply:
You actually introduced a typo here. 
`//Note that typing information ... is lost`